### PR TITLE
YibanChen milestone 3

### PR DESCRIPTION
**Pull request by davidrhodus**
# Milestone Delivery Checklist

- [x] The [milestone-delivery-template.md](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/deliveries/milestone-delivery-template.md) has been copied and updated.
- [x] The [invoice form :pencil:](https://forms.gle/LSRr7PCjBpEbKGh89) has been filled out for this milestone.
- [x] This pull request is being made by the same account as the accepted application.
- [x] In case of acceptance, the payment will be transferred to the BTC/ETH payment address provided in the application.
- [x] The delivery is according to the [Guidelines for Milestone Deliverables](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).

Link to the application pull request: https://github.com/w3f/Grants-Program/pull/499 < please fill this in with the PR number of your application.

### CODE REVIEW

**Comment by alxs** [comment]
Thanks for the delivery @davidrhodus. We'll look into it as soon as possible. Please excuse a longer than usual response time as we currently have a large number of deliveries pending evaluation.

File: deliveries/yibanchen_milestone_3.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Milestone Delivery :mailbox:
+
+**The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).**
+
+- **Application Document:** [YibanChen Application](https://github.com/w3f/Grants-Program/blob/master/applications/yiban_chen1.md)
+- **Milestone Number:** milestone #2
```
Comment (alxs) [inline deliveries/yibanchen_milestone_3.md:6]:
```suggestion
- **Milestone Number:** 3
```

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks again for the delivery. But the [name service pallet](https://github.com/YibanChen/yc-node/blob/9d0eb16ac10c25183c6982c93ab9040c64dfd495/pallets/name-service/src/lib.rs) which you link to is just the Nicks Module developed by parity as far as I can see it! What you described in your application was “The next feature, Name Service, specifically the Name-Service-pallet. YibanChen will leverage existing code from the identity-pallet to start building a substrate blockchain based name service. While we intend for this service to act as a federated login provider. This service could act as a possible replacement for current DNS infrastructure, we also expect this service/protocol could enable name resolution of multiple blockchain objects such as Wallets, NFT's, and possible directory listing to discover different substrate blockchains.” where as the nicks module specifically says: “It makes no effort to create a name hierarchy, be a DNS replacement or provide reverse lookups.” Could you comment on this? Also as far as I can see, you haven’t updated any [documentation](https://github.com/YibanChen/yc-node/blob/site-m2/docs/site-test-guide.md) or inline documentation.

**Comment by Noc2** [comment]
I’m closing the PR due to inactivity. Let me know if you want to continue to work on this. Otherwise we will terminate the grant in two weeks from now.

**Comment by Noc2** [comment]
Fyi: We will go ahead and terminate the grant.

**Pull request closed without merging**